packages/core/src/ceramic.ts

```diff
@@ -175,11 +177,12 @@ export class Ceramic implements CeramicApi {
   public readonly dispatcher: Dispatcher
   public readonly loggerProvider: LoggerProvider
   public readonly pin: PinApi
+  public readonly index: IndexClientApi
```

Comment:
`IndexClientApi` sounds to me like what `RemoteIndexApi` actually is.  Can we rename `IndexClientApi` to `IndexApi` and `IndexApi` to `ServerIndexApi` or `LocalIndexApi`?

Reply:
good idea.